cmd/ocm/delete/cmd.go

```diff
@@ -40,14 +40,32 @@ var args struct {
 }
 
 var Cmd = &cobra.Command{
-	Use:       "delete [flags] PATH",
+	Use:       "delete [flags] PATH|RESOURCE_ALIAS(with resource_id)",
 	Short:     "Send a DELETE request",
 	Long:      "Send a DELETE request to the given path.",
 	RunE:      run,
 	ValidArgs: urls.Resources(),
 }
 
+var arguements_desc_list = map[string]string{
+	"account": `# Delete an account with ACCOUNT_ID
```

Comment:
Think it would be worth to also document the shorter versions of the arguments

Reply:
I don't think we need to document every example invocation in the usage. That should belong in the examples section.

In the usage section we should only document the allowed RESOURCE_ALIAS values